airflow/www/views.py

```diff
@@ -1849,6 +1927,7 @@ def _mark_task_instance_state(  # pylint: disable=too-many-arguments
 
         response = self.render_template(
             "airflow/confirm.html",
+            endpoint=url_for(f'Airflow.{state}'),
```

Comment:
This would be a bit difficult to refactor in the future (won't be found when searching for `Airflow.success` globally); I'd use a plain if else of a dict look up.